notebooks/segmentation_masks.ipynb

```diff
@@ -0,0 +1,158 @@
+{
```

Comment:
Sorry again. moved gen_masks.py to deprecated in the previous PR. We could delete this?

Reply:
Yes! It was just a notebook I found in the repo, but if nobody's using it let's delete it.